packages/arb-token-bridge-ui/src/components/common/SearchPanel/SearchPanelTable.tsx

```diff
@@ -0,0 +1,53 @@
+import { MagnifyingGlassIcon } from '@heroicons/react/24/outline'
+import React, { PropsWithChildren } from 'react'
+
+type SearchPanelTableProps = {
+  searchInputPlaceholder: string
+  searchInputValue: string
+  SearchFieldCta?: React.JSX.Element
+  onSearchInputChange: React.ChangeEventHandler<HTMLInputElement>
+  onSubmit?: React.FormEventHandler<HTMLFormElement>
+  errorMessage: string
+}
+
+export const SearchPanelTable = ({
+  searchInputPlaceholder,
+  searchInputValue,
+  SearchFieldCta,
```

Comment:
NIT: I understand why we're uppercasing this prop-name, but can we do without it from a consistency pov?

Reply:
IMO it's more clear if it's uppercase, then we know it's a component. Otherwise I would expect it to be just some text